diagnostic.md

```diff
@@ -32,23 +36,29 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+1). Getters retrieve a piece of information from the selection.
+2). Getters operate only on the first element in a selection
 
+1). Setters operate on all elements in a selection, using what's known as implicit
+iteration.
+2). Setters alter the selection in some way.
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a visual tree of how your HTML structure looks like, developers can
+build documents, navigate their structure, and add, modify, or delete elements and content.
 <!-- your answer ends here -->
```

Comment:
Nail on the head with visual tree, but try to remember that it is represented by the document object, it doesn't build documents.